spec/plans/results.fmf

```diff
@@ -20,6 +20,12 @@ description: |
        # String, name of the test.
        name: /test/one
 
+       # fmf ID of the test.
+       fmf_id:
+         url: http://some.git.host.com/project/tests.git
+         ref: main
```

Comment:
Let's drop 'ref' line. Very likely it will not be stored as that is the default branch.
Otherwise also 'path' should be added into the example.

Reply:
I'm not sure about it, but I recall seeing `ref: main` in my experiments, even though it's the default branch. I'll add `path` though.

Reply:
True, but the minimal omits keys that are not set, i.e. set to `None`, and it’s used in this patch.

Reply:
Added `path` in 6146e562. I dropped `ref` from the example.

Reply:
@happz seems tests still fail